power/nut-client/pkg.yaml

```diff
@@ -7,15 +7,15 @@ dependencies:
   - image: "{{ .PKGS_PREFIX }}/util-linux:{{ .BUILD_ARG_PKGS }}"
 steps:
   - sources:
-      - url: https://github.com/networkupstools/nut/releases/download/v{{ .NUT_VERSION }}/nut-{{ .NUT_VERSION }}.tar.gz
+      - url: https://github.com/networkupstools/nut/releases/download/v{{ .NUT_VERSION }}-signed/nut-{{ .NUT_VERSION }}.tar.gz
```

Comment:
they re-released 2.8.0 and named it `-signed` for a stable one and normal one without `-signed` is now marked as pre-release for windows :shrug: